ml/jml/probabilizer.h

```diff
@@ -332,6 +336,31 @@ operator << (DB::Store_Writer & store, const GLZ_Probabilizer & prob);
 DB::Store_Reader &
 operator >> (DB::Store_Reader & store, GLZ_Probabilizer & prob);
 
+/*****************************************************************************/
+/* PROBABILIZER                                                          */
+/*****************************************************************************/
```

Comment:
OK, it's much better now it's a pure POD structure.  The last thing I would consider is using a free function to train it rather than a member function, but not a big deal.